spec/models/user_spec.rb

```diff
@@ -8,4 +8,8 @@
     it { should validate_presence_of(:name) }
     it { should validate_presence_of(:date_of_birth) }
   end
+
+  describe 'relationships' do
+    it { should have_many(:executors).dependent(:destroy) }
```

Comment:
Is this testing for multiple executors? I think to start we should only mandate a user chooses one executor upon account creation.

Reply:
It's more testing that a user _may_ have multiple executors. The way it's set up, a user may have zero, one, or many. This way if a user wants to designate more than one (good idea in case they're travelling together and both crash) in future they're able to.

Reply:
Awesome! Thanks.